_includes/misc/advcode-shortcuts.md

```diff
@@ -0,0 +1,13 @@
+## Advanced Code Editor search and replace keyboard shortcuts
+
+Advanced Code Editor provides the following shortcuts for search and replace within the code dialog.
+
+| Action | PC | Mac |
+| ---- | ----- | ----- |
+| Find | Ctrl+F | Cmd+F |
+| Find next instance | Ctrl+G | Cmd+G |
+| Find previous instance | Shift+Ctrl+G | Shift+Cmd+G |
+| Replace | Ctrl+H | Cmd+H |
+| Replace All | Shift+Ctrl+R | Shift+Cmd+Option+F |
+
+For a list of shortcuts available in Advanced Code Editor, see: [CodeMirror - Sublime Text bindings demo](https://codemirror.net/demo/sublime.html).
```

Comment:
Not sure we should be linking this... as nowhere do we say it's just a wrapped CodeMirror instance and that's normally kept hidden?

Reply:
yeah, wasn't sure either. I'll remove that.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1594/commits/42b7fcf5c90819e237ab7dccd63ff44883971ee0